rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -751,10 +740,6 @@ rmw_publisher_assert_liveliness(const rmw_publisher_t * publisher)
   auto pub_data = node_data->get_pub_data(publisher);
   RMW_CHECK_ARGUMENT_FOR_NULL(pub_data, RMW_RET_INVALID_ARGUMENT);
 
-  if (!pub_data->liveliness_is_valid()) {
-    return RMW_RET_ERROR;
-  }
-
```

Comment:
Why remove this?

Reply:
As stated above, the `z_check` has been removed.

Reply:
https://github.com/ZettaScaleLabs/rmw_zenoh/commit/62ca8225256782d57ac376c826975459aea9b417